controllers/account/account_controller.go

```diff
@@ -776,106 +829,106 @@ func TagAccount(awsSetupClient awsclient.Client, awsAccountID string, shardName
-	// created in child accounts
-	roleArn := config.GetIAMArn(currentAcctInstance.Spec.AwsAccountID, config.AwsResourceTypeRole, roleToAssume)
-
-	// Use the role session name to uniquely identify a session when the same role
-	// is assumed by different principals or for different reasons.
-	var roleSessionName = "awsAccountOperator"
-
-	var creds *sts.AssumeRoleOutput
-	var credsErr error
-
-	for i := 0; i < 10; i++ {
-
-		// Get STS credentials so that we can create an aws client with
-		creds, credsErr = getSTSCredentials(reqLogger, awsSetupClient, roleArn, "", roleSessionName)
-		if credsErr != nil {
-			// Get custom failure reason to update account status
-			reason := ""
-			if aerr, ok := credsErr.(awserr.Error); ok {
-				reason = aerr.Code()
-			}
-			errMsg := fmt.Sprintf("Failed to create STS Credentials for account ID %s: %s", currentAcctInstance.Spec.AwsAccountID, credsErr)
-			_, stateErr := r.setAccountFailed(
-				reqLogger,
-				currentAcctInstance,
-				awsv1alpha1.AccountClientError,
-				reason,
-				errMsg,
-				AccountFailed,
-			)
-			if stateErr != nil {
-				reqLogger.Error(stateErr, "failed setting account state", "desiredState", AccountFailed)
-			}
-			return nil, nil, credsErr
-		}
-
-		// If this is a BYOC account, check that BYOCAdminAccess role was the one used in the AssumedRole.
-		// RoleID must exist in the AssumeRoleID string. This is an eventual consistency work-around code.
-		// It can take some varying amount of time to use the correct role if it had just been created.
-		match, _ := matchSubstring(ccsRoleID, *creds.AssumedRoleUser.AssumedRoleId)
-		if ccsRoleID != "" && !match {
-			reqLogger.Info(fmt.Sprintf("Assumed RoleID:Session string does not match new RoleID: %s, %s", *creds.AssumedRoleUser.AssumedRoleId, ccsRoleID))
-			reqLogger.Info(fmt.Sprintf("Sleeping %d seconds", i))
-			time.Sleep(time.Duration(i) * time.Second)
-		} else {
-			break
-		}
-	}
-
-	var awsRegion string
-	if region != "" {
-		awsRegion = region
-	} else {
-		awsRegion = config.GetDefaultRegion()
-	}
-
-	awsAssumedRoleClient, err := r.awsClientBuilder.GetClient(controllerName, r.Client, awsclient.NewAwsClientInput{
-		AwsCredsSecretIDKey:     *creds.Credentials.AccessKeyId,
-		AwsCredsSecretAccessKey: *creds.Credentials.SecretAccessKey,
-		AwsToken:                *creds.Credentials.SessionToken,
-		AwsRegion:               awsRegion,
-	})
-	if err != nil {
-		logger.Error(err, "Failed to assume role")
-		reqLogger.Info(err.Error())
-		errMsg := "Message Failed creating AWS Client with Assumed Role"
-		_, stateErr := r.setAccountFailed(
-			reqLogger,
-			currentAcctInstance,
-			awsv1alpha1.AccountClientError,
-			"AWSClientCreationFailed",
-			errMsg,
-			AccountFailed,
-		)
-		if stateErr != nil {
-			reqLogger.Error(err, "failed setting account state", "desiredState", AccountFailed)
-		}
-		return nil, nil, err
-	}
-
-	return awsAssumedRoleClient, creds, nil
-}
+//func AssumeRoleFunc(r *AccountReconciler,
```

Comment:
Can we remove these lines instead of comment them out? Git history will keep these for us.